frontend/server/src/DAO/CourseIdentityRequest.php

```diff
@@ -12,4 +12,72 @@
  * @access public
  */
 class CourseIdentityRequest extends \OmegaUp\DAO\Base\CourseIdentityRequest {
+    /**
+     * @return list<array{accepted: bool|null, admin_id: int, identity_id: int, last_update: null|string, course_id: int, request_time: string}>|null
```

Comment:
```suggestion
     * @return list<array{accepted: bool|null, admin_id: int, identity_id: int, last_update: null|string, course_id: int, request_time: string}>
```

(ve el comentario de abajo)

Reply:
Se quita el null